offchain/state-server/Cargo.toml

```
@@ -9,8 +9,9 @@ name = "cartesi-rollups-state-server"
 path = "src/main.rs"
 
 [dependencies]
+log = { path = "../log" }
```

Keeping them in alphabetic order:

```suggestion
clap = { workspace = true, features = ["derive", "env"] }
```

---

Done at 9e1b6f8e7bc477c334f4e109cf3bb7a1642bc44d